indico/modules/users/blueprint.py

```diff
@@ -92,6 +92,7 @@
 # Users API
 _bp.add_url_rule('!/api/user/', 'authenticated_user', RHUserAPI)
 _bp.add_url_rule('!/api/affiliations/', 'api_affiliations', RHSearchAffiliations)
+_bp.add_url_rule('!/api/v2/instance', 'foo_bar', RHFooBar, methods=('GET',))  # TODO: Remove this!!! (testing only)
```

Comment:
Tip, for such test code it's much more convenient to add it like this:

```py
@_bp.route('!/api/v2/instance')
def test():
    from flask import jsonify
    return jsonify(foo='bar'), 200
```

Completely self-contained and just one block of code to remove later.